tests/BrowscapTest/Filter/FullFilterTest.php

```diff
@@ -55,77 +67,44 @@ public function testGetType() : void
      */
     public function testIsOutput() : void
     {
-        $division = $this->createMock(\Browscap\Data\Division::class);
+        $division = $this->createMock(Division::class);
 
         self::assertTrue($this->object->isOutput($division));
     }
 
     /**
-     * Data Provider for the test testIsOutputProperty
+     * @group filter
+     * @group sourcetest
      *
-     * @return array<string|boolean>[]
+     * @param mixed $propertyName
+     * @param mixed $isExtra
      */
-    public function outputPropertiesDataProvider()
+    public function testIsOutputProperty() : void
     {
-        return [
```

Comment:
Why was this data provider removed? Are these values checked elsewhere?

Reply:
This function returns `true` independend of the value of the parameters. So I dont think we need a test provider.